rho/clicommands.py

```diff
@@ -266,7 +266,8 @@ def _create_ping_inventory(profile_ranges, profile_auth_list, forks):
 
         out = out.split('\n')
 
-        for line in enumerate(out):
+        # pylint: disable=consider-using-enumerate
```

Comment:
The original code used `range(len(out))` before my lint changes. I encountered an error when running the actual `rho` command with this so I backed out the change for the time being.